InnerEye/ML/configs/histo_configs/classification/DeepSMILEPanda.py

```diff
@@ -109,9 +117,29 @@ def get_data_module(self) -> PandaTilesDataModule:
             cross_validation_split_index=self.cross_validation_split_index,
         )
 
-    def get_trainer_arguments(self) -> Dict[str, Any]:
-        # These arguments will be passed through to the Lightning trainer.
-        return {"callbacks": self.callbacks}
+    def create_model(self) -> DeepMILModule:
+        self.data_module = self.get_data_module()
+        # Encoding is done in the datamodule, so here we provide instead a dummy
+        # no-op IdentityEncoder to be used inside the model
+        self.slide_dataset = self.get_slide_dataset()
+        self.level = 1
+        return DeepMILModule(encoder=IdentityEncoder(input_dim=(self.encoder.num_encoding,)),
+                             label_column=self.data_module.train_dataset.LABEL_COLUMN,
+                             n_classes=self.data_module.train_dataset.N_CLASSES,
+                             pooling_layer=self.get_pooling_layer(),
+                             class_weights=self.data_module.class_weights,
+                             l_rate=self.l_rate,
+                             weight_decay=self.weight_decay,
+                             adam_betas=self.adam_betas,
+                             slide_dataset=self.get_slide_dataset(),
+                             tile_size=self.tile_size,
+                             level=self.level)
+
+    def get_slide_dataset(self) -> Dataset:
+        return Dataset(PandaDataset(root=self.extra_local_dataset_paths[0]))                    # type: ignore 
```

Comment:
why the type is ignored and why you need to wrap the output in a Monai Dataset? PandaDataset is a SlideDataset

Reply:
I could investigate it further if you want but this is how it was used previously and I reused the  way of instantiation.